force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +425,19 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
+        List<Person> people = new List<Person>();
 
         // Loop through each name in the provided list of names
             // Create a new Person object for each name
-            
             // Add the new Person object to the list of Person objects
         
+        for (Integer i=0; i<names.size(); i++) {
+            Person p = new Person(names[i], age);
+            people.add(p);
+        }
         
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
+        return people;
```

Comment:
Totally valid solution. Mine is below. You don't really need that Person variable in the loop

```
public static List<Person> createMultiplePersons(List<String> names, Integer age) {

        List<Person> persons = new List<Person>();
        for (String name : names) {
            persons.add(new Person(name, age));
        }
        return persons;
    }
    ````